sdk/metricsadvisor/tests.yml

```diff
@@ -4,6 +4,7 @@ extends:
   template: ../../eng/pipelines/templates/stages/archetype-sdk-tests.yml
   parameters:
     ServiceDirectory: metricsadvisor
+    TimeoutInMinutes: 90
```

Comment:
Increasing timeout because we were getting lots of "The job running on agent (...) ran longer than the maximum time of 60 minutes" errors.